minard/ecaldb.py

```diff
@@ -0,0 +1,115 @@
+import sqlalchemy
+from .db import engine
+import time
+
+penn_daq_tests = {
```

Comment:
minor issue, but I try to name all global variables uppercase, so PENN_DAQ_TESTS